crates/bitwarden/src/auth/client_auth.rs

```diff
@@ -1,15 +1,27 @@
-use super::{
-    password::{password_strength, satisfies_policy, MasterPasswordPolicyOptions},
-    register::{make_register_keys, register},
-    RegisterKeyResponse, RegisterRequest,
+#[cfg(feature = "secrets")]
+use crate::auth::login::{login_access_token, AccessTokenLoginRequest, AccessTokenLoginResponse};
+use crate::{auth::renew::renew_token, error::Result, Client};
+#[cfg(feature = "internal")]
+use crate::{
+    auth::{
+        login::{
+            login_api_key, login_password, send_two_factor_email, ApiKeyLoginRequest,
+            ApiKeyLoginResponse, PasswordLoginRequest, PasswordLoginResponse,
+            TwoFactorEmailRequest,
+        },
+        password::{password_strength, satisfies_policy, MasterPasswordPolicyOptions},
+        register::{make_register_keys, register},
+        RegisterKeyResponse, RegisterRequest,
+    },
+    client::kdf::Kdf,
 };
-use crate::{client::kdf::Kdf, error::Result, Client};
 
 pub struct ClientAuth<'a> {
     pub(crate) client: &'a mut crate::Client,
 }
 
 impl<'a> ClientAuth<'a> {
+    #[cfg(feature = "internal")]
```

Comment:
I'm wondering if it would be easier for us to just have two `impl<'a> ClientAuth<'a> {` blocks and put the feature flag directly on the block instead of repeating it all over the place. Not really necessary for this PR though